spec/bcm2835_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'spec_helper'
+
+
+describe 'Bcm2835' do
+  
+  before :context do
+    Platform.driver
+  end
+  
+  after :context do
+    Bcm2835.exported_pins.delete_if { true }
+  end
+
+  # describe  'GPIO' do
+    let(:file_like_object) { double("file like object") }
+
+    before :example do
+      allow(File).to receive(:read).and_return("1")
+      allow(File).to receive(:write).and_return("1")
+      allow(File).to receive(:open).and_return(file_like_object)
+    end
+
+    it '#export' do
```

Comment:
Sure, that's fair but I think we should keep a good level of quality :smile: 
I suppose we can leave it like this and then go ahead and fix them after `2.0.0`.